src/mono/wasm/runtime/library_mono.js

```diff
@@ -779,6 +779,64 @@ var MonoSupportLib = {
 			// and nested class names like Foo/Bar to Foo.Bar
 			return className.replace(/\//g, '.').replace(/`\d+/g, '');
 		},
+
+		wasm_load_timezone_data: function (file_name) {
```

Comment:
this doesn't need to be timezone specific.  I think this should become something like mono_wasm_load_data where it takes the array as the first arg and possibly a second prefix arg and them move the read() call to the test runner before it calls this.